auth.go

```diff
@@ -0,0 +1,128 @@
+package nutshttp
+
+import (
+	"errors"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/gin-gonic/gin"
```

Comment:
Imports are organized in groups, with blank lines between them. The standard library packages are always in the first group. see **https://github.com/golang/go/wiki/CodeReviewComments#imports**